jsonalchemy/wrappers.py

```diff
@@ -60,6 +60,14 @@ def __init__(self, schema=None, root=None):
         else:
             self.root = self
 
+    def search(self, query):
+        jsonpath_expr = parse(query)
+        result = [match.value for match in jsonpath_expr.find(self)]
+
+        return wrap(result,
+                    {'type': 'array', 'items': [el.schema for el in result]},
+                    result)
```

Comment:
The implementation of this line changed here:

https://github.com/MSusik/jsonalchemy/commit/15a37c52fb0772147ebaced2270165e2cf1d50be?diff=split#diff-e33cba0b3eaa91fdddd2c9faa1ae5dd7R74

but the question is still valid.